Jelly/Classes/private/Animators/Animated Animators/CoverAnimator.swift

```diff
@@ -56,6 +57,19 @@ extension CoverAnimator : UIViewControllerAnimatedTransitioning {
         let propertyAnimator = UIViewPropertyAnimator(duration: animationDuration, timingParameters: timingCurveProvider)
 
         propertyAnimator.addAnimations {
+            if isPresentation {
```

Comment:
It would be nice if this code will only be executed if the depthscale is other than 1.0. 
If we do not need to transform the presentingViewController we also don't need to create a masklayer etc.

Reply:
Will update